eZ/Bundle/EzPublishCoreBundle/Command/UpdateTimestampsToUTCCommand.php

```diff
@@ -0,0 +1,538 @@
+            $this->timezone = $this->validateTimezone($timezone, $output);
+
+            $modeTxt = 'Converting timestamps for both ezdate and ezdatetime fields.';
+
+            switch ($this->mode) {
+                case 'date':
+                    $modeTxt = 'Converting timestamps for ezdate fields.';
+                    break;
+                case 'datetime':
+                    $modeTxt = 'Converting timestamps for ezdatetime fields.';
+                    break;
+            }
+
+            $output->writeln(
+                [
+                    $modeTxt,
+                    'Calculating number of Content Object Attributes to update...',
+                ]
+            );
+            $count = $this->countEzDateObjectAttributes();
+            $output->writeln(
+                [
+                    sprintf('Found total of Content Objects Atributes for update: %d', $count),
+                    '',
+                ]
+            );
+
+            if ($count == 0) {
+                $output->writeln('Nothing to process, exiting.');
+
+                return;
+            }
+
+            $helper = $this->getHelper('question');
+            $question = new ConfirmationQuestion(
+                '<question>Are you sure you want to proceed?</question> ',
+                false
+            );
+
+            if (!$helper->ask($input, $output, $question)) {
+                $output->writeln('');
+
+                return;
+            }
+
+            $progressBar = $this->getProgressBar($count, $output);
+            $progressBar->start();
+
+            for ($offset = 0; $offset < $count; $offset += $iterationCount) {
+                $processBuilder = new ProcessBuilder(
+                    [$this->getPhpPath(), $consoleScript, $this->getName(), $this->timezone]
+                );
+
+                if ($from) {
+                    $processBuilder->add('--from=' . $from);
+                }
+                if ($to) {
+                    $processBuilder->add('--to=' . $to);
+                }
+                $processBuilder->add('--mode=' . $this->mode);
+                $processBuilder->add('--offset=' . $offset);
+                $processBuilder->add('--iteration-count=' . $iterationCount);
+                $processBuilder->add('--env=' . $input->getOption('env'));
+                if ($this->dryRun) {
+                    $processBuilder->add('--dry-run');
+                }
+                $processBuilder->setEnv('INNER_CALL', 1);
+                $process = $processBuilder->getProcess();
+                $process->run();
+
+                if (!$process->isSuccessful()) {
+                    throw new RuntimeException($process->getErrorOutput());
+                }
+
+                $this->done += (int) $process->getOutput();
+
+                $progressBar->advance($this->done);
+            }
+
+            $progressBar->finish();
+            $output->writeln(
+                [
```

Comment:
This goes for several places here, but you can safely place the array start on line of the call when it's the only argument.

Double check with php-cs-fixer but at least before I don't think it minds.

so:
```
            $output->writeln([
                '',// Empty newline
                sprintf('Done: %d', $this->done),
            ]);
```